sci-geosciences/laszip/laszip-3.4.4.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake-multilib
+
+DESCRIPTION="Library for free and lossless compression of the LAS LiDAR format"
+HOMEPAGE="https://rapidlasso.de/"
+
+MY_PN="LASzip"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/LASzip/LASzip.git"
+else
+	SRC_URI="https://github.com/${MY_PN}/${MY_PN}/releases/download/${PV}/${PN}-src-${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
If the upstream distfile name is fine, we shouldn't rename it, as it unnecessarily harms provenance.